fix default error path

jo-eman opened the pull request.

jo-eman committed:
fix default error path

bomanviktor reviewed (changes requested):
Remove test(0).txt and squash the commits

bomanviktor committed:
Move `default_errors` into `files`

- Add all error codes

- Change default path to `/files/default_errors` in `check_errors`.

- Remove `SERVER` header

bomanviktor committed:
Change multiple error codes

Update unit tests for invalid method

Remove to_bytes and replace with Bytes::from

bomanviktor reviewed (approved):
LGTM
